taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs

```diff
@@ -202,13 +202,13 @@ private void CreateLoopCondition(object sender, EventArgs e)
             var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginLoopCommand).ToList();
 
             frmCommandEditor editor = new frmCommandEditor(automationCommands, null);
-            editor.selectedCommand = new BeginLoopCommand();
+            editor.SelectedCommand = new BeginLoopCommand();
             var res = editor.ShowDialog();
 
             if (res == DialogResult.OK)
             {
                 //get data
-                var configuredCommand = editor.selectedCommand as BeginLoopCommand;
+                var configuredCommand = editor.SelectedCommand as BeginLoopCommand;
                 var displayText = configuredCommand.GetDisplayValue();
                 var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(configuredCommand);
```

Comment:
Could likely be reduced to JsonConvert.SerializeObject

Reply:
As you know, we are not cleaning up Command Files right now.
It's just updating variable names in dependent files.